splinterd/packaging/splinterd.toml.example

```diff
@@ -168,32 +168,111 @@ version = "1"
+# splinterd verbose option (-v, --verbose) will increase the logging level on
+# this appender.
+#
 #[appenders.stdout]
-
+#
 # "kind" options are stdout,stderr,file,rolling_file
 #kind = "stdout"
+#
 # Pattern controls the formatting of each log message.
 #pattern = "[ {d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
 
-#[appenders.rolling_file]
-
+# The debuglog appender is an example of a rolling_file appender. It creates
+# a log file with debug and higher messages.
+#
+#[appenders.debuglog]
+#
 #kind = "rolling_file"
-# Filename is specific to rolling_file and file appenders and is simply a path to a file.
-# Must be specified.
-#filename = "/var/log/splinter/splinterd.log"
-# Size is specific to rolling_file and specifies when the file should be rolled/overwritten.
-# Must be specified.
+#
+# Pattern controls the formatting of each log message.
+#pattern = "[ {d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
+#
+# Filename is specific to rolling_file and file appenders and is simply a path
+# to a file.  Must be specified.
+#filename = "/var/log/splinter/splinterd-debug.log"
+#
+# Size is specific to rolling_file and specifies when the file should be
+# rolled/overwritten.  Must be specified.
 #size = "16.0M"
+#
+# Level on an appender specifies which messages should be sent to the appender;
+# in this case, all messages level Debug or higher will be sent to the
+# appender.
+#level = "Debug"
+
+# The warnlog appender is the same as the debuglog appender, but with a
+# different level value. This appender can be used in combination with the
+# debuglog appender.
+#
+#[appenders.warnlog]
+#
+#kind = "rolling_file"
+#
+# Pattern controls the formatting of each log message.
+#pattern = "[ {d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
+#
+# Filename is specific to rolling_file and file appenders and is simply a path
+# to a file.  Must be specified.
+#filename = "/var/log/splinter/splinterd-warn.log"
+#
+# Size is specific to rolling_file and specifies when the file should be
+# rolled/overwritten.  Must be specified.
+#size = "16.0M"
+#
+# All messages "Warn" and higher will be sent this appender.
+#level = "Warn"
+
+# The root logger ([loggers.root]) section specifies defaults for message
+# generation.
+#
+#[loggers.root]
+#
+# By default, messages will be sent only to the "stdout" appender. Uncomment
+# the second example to send messages to the appenders defined above (or add
+# your own appenders here).
+#appenders = [ "stdout"]
+#appenders = [ "stdout", "debuglog", "warnlog"]
+#
+# This is the default level at which messages are generated. Messages are
+# generated by loggers and filtered by appenders, which means if you want
+# "Debug" messages to appear, you must set the level here to "Debug" (or lower)
+# and also set the level for the appender to "Debug" (or lower).
+#
+# Valid options are, Trace, Debug, Info, Warn, and Error in decreasing order of
+# verbosity.
+#level = "Warn"
```

Comment:
This should be '#level = "Trace"' since that's the default value.

Reply:
fixed